src/encord_active/lib/coco/encoder.py

```diff
@@ -256,33 +257,25 @@ def get_image(self, label_hash: str, data_unit: dict) -> dict:
         data_hash = data_unit["data_hash"]
         self._data_hash_to_image_id_map[(data_hash, 0)] = image_id
         return {
-            "coco_url": data_unit["data_link"],
+            "coco_url": "",
             "flickr_url": "",
             "id": image_id,
             "image_title": data_unit["data_title"],
-            "file_name": self.get_file_name_and_download_image(label_hash, data_unit),
+            "file_name": self.get_local_path_or_file_name(data_unit),
             "height": data_unit["height"],
             "width": data_unit["width"],
             "label_hash": label_hash,
             "data_hash": data_hash,
             "frame_num": int(data_unit["data_sequence"]),
         }
 
-    def get_file_name_and_download_image(self, label_hash: str, data_unit: dict) -> str:
-        data_hash = data_unit["data_hash"]
-        url = data_unit["data_link"]
-        image_extension = data_unit["data_type"].split("/")[-1]
-
-        relative_destination_path = Path("data").joinpath(
-            Path(label_hash), Path("images"), Path(f"{data_hash}.{image_extension}")
-        )
-        absolute_destination_path = self._download_file_path.joinpath(relative_destination_path)
-        download_condition = self._download_files and not absolute_destination_path.exists()
-
-        if download_condition or self._force_download:
-            self.download_image(url, absolute_destination_path)
-
-        return str(relative_destination_path)
+    def get_local_path_or_file_name(self, data_unit: dict) -> str:
+        url: str = data_unit["data_link"]
```

Comment:
nit: double check this for local files: the database row is the source of truth for filepaths for the program.